content/client-lib-development-guide/features.textile

```diff
@@ -378,14 +384,19 @@ h3(#realtime-channel). Channel
 * @(RTL11)@ If a channel enters the @DETACHED@ or @FAILED@ state, then all messages that are still queued for send on that channel should be deleted from the queue triggering a failure for the publish or presence methods invoked for those messages
 * @(RTL4)@ @Channel#attach@ function:
 ** @(RTL4a)@ If already @ATTACHED@ or @ATTACHING@ nothing is done
+** @(RTL4h)@ If the channel is in the @DETACHING@ state, do the operation once it's @DETACHED@.
+** @(RTL4g)@ If state is @FAILED@, the @attach@ request results in an error
```

Comment:
@tcard was just about to merge this, but then noticed this.  This is wrong no, if you call @attach@ on a @FAILED@ channel it will initiate a new @ATTACH@ no?